contrib/glycin-loaders/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "glycin-loaders"
+pkgver = "1.0.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dtests=false"]
+hostmakedepends = [
+    "cargo",
+    "gettext",
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "cairo-devel",
+    "libheif-devel",
+    "libjxl-devel",
+    "libseccomp-devel",
+    "libxml2-devel",
+    "pango-devel",
+    "rust-std",
+]
+depends = ["bubblewrap"]
+pkgdesc = "Sandboxed and extendable image decoding"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "MPL-2.0 OR LGPL-2.1-or-later"
+url = "https://gitlab.gnome.org/sophie-h/glycin"
+source = f"$(GNOME_SITE)/{pkgname}/{pkgver[:-2]}/{pkgname}-{pkgver}.tar.xz"
+sha256 = "d0f022462ff555856e85ea940474470bb36b37c9ffcbcba63a03fe5e954370cf"
+# Needs loaders to be system-installed
+options = ["!check"]
+
+
+def init_build(self):
+    from cbuild.util import cargo
+
+    renv = cargo.get_environment(self)
+    self.make_env.update(renv)
+
+
+def post_install(self):
+    for loader in [
+        "glycin-heif",
+        "glycin-image-rs",
+        "glycin-jxl",
+        "glycin-svg",
+    ]:
```

Comment:
Do you think it'd be fine here to dynamically find the loaders that were built, so that we don't miss something added in the future inadvertently? If so, how can I do that?

Reply:
i think iterating on filepaths and checking for executable && !directory in target/release lets you do that